autoload/editorconfig_core/fnmatch.vim

```diff
@@ -0,0 +1,441 @@
+" encoding is utf8 - 中文
```

Comment:
Is there a reason to have Chinese characters on this line?

Reply:
It is my personal style, but I can remove it if you wish.  I like to include some non-ASCII characters on the first line so that programs can more easily tell it's UTF-8 text.

Reply:
How about [IDEOGRAPHIC FULL STOP](http://www.fileformat.info/info/unicode/char/3002/index.htm)? It just looks semantically less confusing.

    encoding is utf8。

Reply:
Thanks!  I like that and have made the change.